next/components/forms/RadioButton/Radio.tsx

```diff
@@ -0,0 +1,123 @@
+import HelpIcon from '@assets/images/forms/icon-help.svg'
+import cx from 'classnames'
+import * as React from 'react'
+import { useRadio } from 'react-aria'
+
+import Tooltip from '../Tooltip'
+import { RadioContext } from './RadioGroup'
+
+type RadioBase = {
+  variant?: 'basic' | 'boxed' | 'card'
+  className?: string
+  isDisabled?: boolean
+  error?: boolean
+  children: React.ReactNode
+  value: string
+  tooltip?: string
+}
+
+const Radio = ({
+  error = false,
+  isDisabled = false,
+  tooltip,
+  variant = 'basic',
+  className,
+  ...rest
+}: RadioBase) => {
+  const state = React.useContext(RadioContext)
+  const [isTooltipOpened, setIsTooltipOpened] = React.useState<boolean>(false)
+  const ref = React.useRef(null)
+  const { inputProps } = useRadio({ ...rest, isDisabled }, state, ref)
+  const inputStyle = cx(
+    `focus-visible:outline-none focus:outline-none appearance-none bg-white m-0 w-6 h-6 grid place-content-center left-0 right-0 top-0 bottom-0 rounded-full border-2 border-solid`,
```

Comment:
don't use ` if not interpolating:

```suggestion
    'focus-visible:outline-none focus:outline-none appearance-none bg-white m-0 w-6 h-6 grid place-content-center left-0 right-0 top-0 bottom-0 rounded-full border-2 border-solid',
```